src/server.js

```diff
@@ -7,6 +7,13 @@ const addEventListeners = (socket) => {
   socket.on('dial', () => {
     socket.emit('created', 'created success');
   });
+  socket.on('sice', (candidate) => {
+    socket.broadcast.to(candidate.room).emit('rice', candidate);
```

Comment:
When ```candidate``` or ```candidate.room``` are undefined, error occurs.

Reply:
I apply your opinion
see [this](https://github.com/the-boyj/signaling-server/pull/27/commits/1483bab032e417eec55aea5c564b8ba224e6d0cf)